ansible/files/postgresql_config/supautils.conf.j2

```diff
@@ -1,4 +1,5 @@
 supautils.extensions_parameter_overrides = '{"pg_cron":{"schema":"pg_catalog"}}'
+supautils.policy_grants = '{"postgres":["auth.audit_log_entries","auth.identities","auth.refresh_tokens","auth.sessions","auth.users","realtime.broadcasts","realtime.channels","realtime.presences","storage.buckets","storage.migrations","storage.objects"]}'
```

Comment:
For Auth, these are tables with policies across all projects - we may want to further restrict the list down the line.